moebot_bot/util/db/poll.go

```diff
@@ -0,0 +1,92 @@
+package db
+
+import "log"
+
+type Poll struct {
+	Id        int
+	Options   []*PollOption
+	Title     string
+	Open      bool
+	UserId    string
+	ChannelId string
+	MessageId string
+}
+
+const (
+	pollTable = `CREATE TABLE IF NOT EXISTS poll(
+		Id SERIAL NOT NULL PRIMARY KEY,
+		Title VARCHAR(100) NOT NULL,
+		ChannelId VARCHAR(20) NOT NULL,
```

Comment:
(I should add this to the readme, or some sort of style guide, my mistake)
In the database I setup a style of `objectUid` to mean discord ID and `objectId` to mean database ID. However, I'd also like to maintain some level of normalization within the database so could we make this foreign keys to the channel table? I should add a user table to further improve this, especially with EXP + profiles coming soon...
I'd be happy to discuss the drawbacks of this if you're not on board though!